server/repository/src/db_diesel/vaccination.rs

```diff
@@ -130,7 +131,11 @@ fn create_filtered_query(filter: Option<VaccinationFilter>) -> BoxedVaccinationQ
     let mut query = vaccination::table
         .left_join(clinician_link::table.inner_join(clinician::table))
         .inner_join(vaccine_course_dose::table)
-        .left_join(name_link::table.inner_join(name::table))
+        .left_join(
```

Comment:
btw, looks like this was never actually used for sorting, sub query would have worked I think